### PR TITLE
Add anvil blog post

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

File: content/english/blog/anvil.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+We've been using [Anvil](https://anvil.works/) for some while now - since we met at PyCon UK 2017 - and we've found it to be a great tool for building web applications.
+
+Since then, we've recommended it to several clients and undertaken projects for various organisations that were already using it.
+
+For us, there are compelling reasons both technical and commercial to use Anvil as our go-to tool for web development and to continue to recommend it to others.
+
+## What is Anvil?
+Anvil is a platform for building web applications.
+
+It's a development framework, a hosting and support service, a community of developers and tool that scales from solo hobby projects to serious enterprise applications.
```
Comment (WelshGreek) [inline content/english/blog/anvil.md]: a tool

File: content/english/blog/anvil.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+However, Anvil also has 'Escape Hatches' - if the Python abstraction is getting in the way, we can drop down to writing our own JavaScript, HTML, CSS or SQL. We're not constrained by the Anvil abstraction - it's there to help make things easy but we can always go around it if we need to.
+
+## Commercial Strengths
+Anvil is a commercial product. That means that there is a company behind it, providing support, maintaining the platform and developing new features.
+
+It also means our clients have contracts with Anvil to provide the hosting and support services that are needed to keep their applications running - leaving us free to concentrate on what we do best - building great applications.
+
+We don't have to be on call for anything unexpected - Anvil's support team are there to help with any issues that arise. Our clients know they can replace us with another developer if they need to - the application is built in a standard Python environment and the code is theirs to keep.
```
Comment (WelshGreek) [inline content/english/blog/anvil.md]: to keep or allow to be shared.....

File: content/english/blog/anvil.md (hunk 1 above)
Comment (WelshGreek) [inline content/english/blog/anvil.md]:
```suggestion
It's a development framework, a hosting and support service, a community of developers and a tool that scales from solo hobby projects to serious enterprise applications.
```

File: content/english/blog/anvil.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Web Development with Anvil"
+date: 2024-07-06T11:13:38+01:00
+categories:
+  - Python
+  - Web Development
+author: Owen
+---
+We've been using [Anvil](https://anvil.works/) for some while now - since we met at PyCon UK 2017 - and we've found it to be a great tool for building web applications.
```
Comment (theref) [inline content/english/blog/anvil.md]: sounds like we (us 3) only met at pycon - maybe mention that they sponsored or something?

File: content/english/blog/anvil.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Since then, we've recommended it to several clients and undertaken projects for various organisations that were already using it.
```
Comment (theref) [inline content/english/blog/anvil.md:11]: "default recommendation"?

File: content/english/blog/anvil.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+For us, there are compelling reasons both technical and commercial to use Anvil as our go-to tool for web development and to continue to recommend it to others.
+
+## What is Anvil?
+Anvil is a platform for building web applications.
```
Comment (theref) [inline content/english/blog/anvil.md]: "full stack"?

File: content/english/blog/anvil.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+It's a development framework, a hosting and support service, a community of developers and a tool that scales from solo hobby projects to serious enterprise applications.
```
Comment (theref) [inline content/english/blog/anvil.md]:
```suggestion
It's a development framework, a hosting and support service, a community of developers, and a tool that scales from solo hobby projects to serious enterprise applications.
```
I'm always a fan of an oxford comma

File: content/english/blog/anvil.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+Anvil has a drag-and-drop designer for building the user interface, a Python editor for writing the code that powers the application and a deployment system that makes it easy to get your application online.
```
Comment (theref) [inline content/english/blog/anvil.md]:
```suggestion
Anvil has a drag-and-drop designer for building the user interface, a Python editor for writing the code that powers the application, and a deployment system to get your application online.
```

File: content/english/blog/anvil.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+But let's also be clear about what Anvil is not - It's not a low-code or a no-code platform. To use it, you need to write Python code.
```
Comment (theref) [inline content/english/blog/anvil.md:22]: Bold some of this - modern blogs use a lot of formatting to get their point across

File: content/english/blog/anvil.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+## Technical Advantages
+Web development is difficult. There are many moving parts, many technologies to learn and many ways to get things wrong.
+
+To counter that, Anvil provides a single Python based environment in which to build an application. We don't need to worry about setting up a server, configuring a database, managing security or any of the other myriad tasks that can distract from building an application.
```
Comment (theref) [inline content/english/blog/anvil.md]:
```suggestion
To counter that, Anvil provides a single Python based environment in which to build an application. We don't need to worry about setting up a server, configuring a database, managing security or any of the other tasks that can distract from building an application.
```

File: content/english/blog/anvil.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+However, Anvil also has 'Escape Hatches' - if the Python abstraction is getting in the way, we can drop down to writing our own JavaScript, HTML, CSS or SQL. We're not constrained by the Anvil abstraction - it's there to help make things easy but we can always go around it if we need to.
```
Comment (theref) [inline content/english/blog/anvil.md]:
```suggestion
However, Anvil also has 'Escape Hatches' - if the Python abstraction is getting in the way, we can drop down to writing our own JavaScript, HTML, CSS or SQL. We're not constrained by the Anvil abstraction - it's there to help make things easy. We can always go around it if we need to.
```

File: content/english/blog/anvil.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+
+## Commercial Strengths
+Anvil is a commercial product. That means that there is a company behind it, providing support, maintaining the platform and developing new features.
```
Comment (theref) [inline content/english/blog/anvil.md]:
```suggestion
Anvil is a commercial product. That means that there is a company behind it, providing support, maintaining the platform, and developing new features.
```
https://www.youtube.com/watch?v=P_i1xk07o4g

**Review by theref — APPROVED** [review]
Small suggestions, none of them are required it's just personal taste

File: content/english/blog/anvil.md (hunk 4 above)
Comment (meatballs) [reply content/english/blog/anvil.md:11]: Not sure I like that. It sounds like we don't bother thinking about a recommendation

**Pull request merged**